stacks/BasicApiStack.ts

```diff
@@ -51,13 +51,18 @@ export function BasicApiStack ({ app, stack }: StackContext): { queue: Queue, bu
           TABLE_NAME: table.tableName,
           QUEUE_URL: queue.queueUrl,
           CLUSTER_BASIC_AUTH_TOKEN: process.env.CLUSTER_BASIC_AUTH_TOKEN ?? '',
-          CLUSTER_IPFS_ADDR: process.env.CLUSTER_IPFS_ADDR ?? ''
+          CLUSTER_IPFS_ADDR: process.env.CLUSTER_IPFS_ADDR ?? '',
```

Comment:
Misconception: What you're setting here is actually the address of ELASTIC IPFS, and to CLUSTER IPFS.
Elastic IPFS is our product running in the cloud.
Cluster IPFS is an on prem setup where they currently store pinned blocks.